build/index.d.ts

```diff
@@ -0,0 +1,17 @@
+declare const ReconnectingWebsocket: (url: string, protocols?: string | string[], options?: {
+    constructor?: {
+        new (url: string, protocols?: string | string[]): WebSocket;
```

Comment:
I see, I'll change the constructor typing so I could be any instantiable function